### PR TITLE
HSEARCH-2079 Improve OSGi header generation with properties and revie…

### PR DESCRIPTION
**Pull request by Sanne**
…w some stale definitions

 - https://hibernate.atlassian.net/browse/HSEARCH-2079

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -156,6 +156,17 @@
...
 
+        <!-- OSGi ranges for some primary dependencies -->
+        <!-- Warning:
+            Ranges are maintained as best effort on our knowledge but only the
+            exact versions in this pom are being tested.
+            Essentially we allow older versions when we know no changes were necessary when we upgraded the dependency. -->
+        <luceneOsgiRangeVersion>[5.3.0,5.5.0)</luceneOsgiRangeVersion>
+        <hibernateOrmOsgiRangeVersion>[5.0.0,5.1.0)</hibernateOrmOsgiRangeVersion>
+        <hibernateHcannOsgiRangeVersion>[5,6)</hibernateHcannOsgiRangeVersion>
+        <hibernateJGroupsOsgiRangeVersion>[3.6.0,4)</hibernateJGroupsOsgiRangeVersion>
```
Comment (gunnarmorling) [inline pom.xml]: Why is it "hibernateJGroups"? Not just "jGroupsOsgiRangeVersion"?

**Comment by gunnarmorling** [comment]
Looks good apart from that one comment.

**Comment by Sanne** [comment]
thanks @gunnarmorling for catching that. Fixed it, I'll push now if the tests pass on CI.

**Comment by Sanne** [comment]
merged

**Pull request merged**